deploy/swrs/utility/schema.sql

```diff
@@ -0,0 +1,8 @@
+-- Deploy ggircs:swrs/utility/schema to pg
+
+begin;
+
+create schema swrs_utility;
```

Comment:
I don't like the word `utility`, it's a bit of a catch-all, meaning that this schema might end up with a variety of unrelated things.
How about `ggircs_parameters`, or `ggircs_settings` since the values stored in this schema will affect more than the SWRS ETL?
Happy to think of other names. This needs to be decided in this PR though, otherwise updating is going to be painful.

Reply:
`ggircs_context`? What has been moved into this schema is contextual data about carbon tax and naics codes